internal/runners/activate/activate.go

```diff
@@ -72,8 +72,7 @@ type primeable interface {
 
 func NewActivate(prime primeable) *Activate {
 	return &Activate{
-		NewNamespaceSelect(prime.Config()),
-		NewCheckout(git.NewRepo(), prime),
```

Comment:
I think we can drop `NewCheckout`?

Reply:
Yes, `NewCheckout` is gone in favor of `checkout.New()`

Reply:
Its use is, but so far as I can tell from the PR diff the function definition wasn't deleted.

Reply:
The file `internal/runners/activate/checkout.go` was deleted. I think the GitHub diff defaults to not actually showing that file, just that it was deleted.